sysl2/sysl/ints.go

```diff
@@ -12,31 +11,54 @@ import (
 
 const endpointWildcard = ".. * <- *"
 
-func GenerateIntegrations(
-	rootModel, title, output, project, filter, modules string,
-	exclude []string,
-	clustered, epa bool,
-) (map[string]string, error) {
+func GenerateIntegrations(intgenParams *CmdContextParamIntgen) (map[string]string, error) {
 	r := make(map[string]string)
 
-	mod, err := loadApp(rootModel, modules)
+	log.Infof("root: %s\n", *intgenParams.root)
+	log.Infof("project: %v\n", *intgenParams.project)
+	log.Infof("clustered: %t\n", *intgenParams.clustered)
+	log.Infof("exclude: %s\n", *intgenParams.exclude)
+	log.Infof("epa: %t\n", *intgenParams.epa)
+	log.Infof("title: %s\n", *intgenParams.title)
+	log.Infof("filter: %s\n", *intgenParams.filter)
+	log.Infof("modules: %s\n", *intgenParams.modules)
+	log.Infof("output: %s\n", *intgenParams.output)
+	log.Infof("loglevel: %s\n", *intgenParams.loglevel)
+
+	if intgenParams.plantuml == nil {
+		plantuml := os.Getenv("SYSL_PLANTUML")
+		intgenParams.plantuml = &plantuml
+		if *intgenParams.plantuml == "" {
+			*intgenParams.plantuml = "http://localhost:8080/plantuml"
```

Comment:
I feel that a warning log to inform local plantuml server is being used will be helpful